src/UI/Factory.php

```diff
@@ -336,4 +337,27 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers are used to display secondary information to a component.
```

Comment:
There is more to that then displaying information. What about: "Popovers can be used if space is scarce to display secondary information such as a preview of some item or to offer limited user interaction."